docs/en/development/build-osx.md

```diff
@@ -37,7 +37,7 @@ sudo xcode-select --install
 
 ``` bash
 brew update
-brew install cmake ninja libtool gettext llvm gcc binutils grep findutils
+brew install ccache cmake ninja libtool gettext llvm gcc binutils grep findutils
```

Comment:
Should we also update `cmake` command below to use it?

Reply:
no need to update other commands